### PR TITLE
version-selector

### LINKED ISSUE #11554: The dropdown list fails to update correctly
### Current State
 The dropdown list fails to update correctly, showing 'latest' instead of the selected version (e.g., 'v0.6').
Screenshot 2024-08-05 023026

### Desired State

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by vaishnaviadhau**
**Notes for Reviewers**

This PR fixes #11554

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 152b1919ad257b53fe736e6db3b49ef33a36bd89 at: https://66d70207a7edd4218570d9ff--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c9a0eaee01ddd4e1dac5a12df0c5df67cb50ba5f at: https://66d71c70064dad4dc0d7382c--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c9a0eaee01ddd4e1dac5a12df0c5df67cb50ba5f at: https://66d71c78d6f7df4f843d8bff--meshery-docs-preview.netlify.app

**Comment by Ashparshp** [comment]
Thank you @vaishnaviadhau,
Noticed an issue: selecting `v6` shows `latest`, and clicking `latest `again still stays on `v6`. Works fine with other versions. Otherwise, looks good to me.

Could you please fix this?

[Meshery Documentation _ Meshery.webm](https://github.com/user-attachments/assets/97f38e31-9b87-41a5-a174-a2fbe27f9f29)

**Comment by vaishnaviadhau** [comment]
@Ashparshp : the changes  are visible when I run the site locally, but they aren’t showing up in the preview section after I raised the PR. Could you check if something is affecting this on the PR side?

File: docs/_includes/header.html [hunk 1]
```diff
@@ -48,10 +48,30 @@
...
             class="nav-item mr-4 mb-2 mb-lg-0 text-black"
           >
-            <span>{{ link.version }}</span>
-          </option>
+         {{ link.version }}
+           </option>
           {% endfor %}
         </select>
+        
+         <script>
```
Comment (sudhanshutech) [inline docs/_includes/header.html:56]: what this script for?

**Review by sudhanshutech — CHANGES_REQUESTED** [review]
I am not seeing and changes in behaviour with the current scenario and the changes from this pr. When i am trying to click 0.6 it selects the latest because 0.6 is the latest.
Either case should be that latest and 0.6 should consolidated at one.

**Pull request closed without merging**